integration-tests/src/test/resources/apps/coherence-proxy-client/src/main/java/cohapp/CacheClient.java

```diff
@@ -0,0 +1,395 @@
+package cohapp;
+
+/*
```

Comment:
Where is this code from?  Needs corrected license.

Reply:
It is almost all new, but I started with some old code, I will fix the license.